frontend/src/scenes/PreflightCheck/preflightLogic.tsx

```diff
@@ -224,7 +224,7 @@ export const preflightLogic = kea<preflightLogicType>([
         siteUrlMisconfigured: [
             (s) => [s.preflight],
             (preflight): boolean => {
-                if (process.env.STORYBOOK) {
+                if (global.process?.env.STORYBOOK) {
```

Comment:
The build was erroring out with "process is undefined" and I really have no idea why this would pop up now.